configure.ac

```diff
@@ -184,10 +185,10 @@ fi
 ##############################################################################
 # GNU Scientific Library
 
-AC_CHECK_FUNC(cblas_cgemm, [], [AC_CHECK_LIB(gslcblas, cblas_cgemm)])
-
-AC_CHECK_LIB(gsl, gsl_sf_bessel_Jn, [],
-	[AC_MSG_WARN([Missing GNU GSL library...Bessel-function field initialization will not be supported.])])
+AC_CHECK_LIB([m],[cos])
+AC_CHECK_LIB([gslcblas],[cblas_dgemm])
```

Comment:
I can't reproduce your problem.  On jdj.mit.edu, `gcc -o foo foo.c -lgsl -lopenblas` works where `foo.c` is
```
#include <math.h>
#include <stdio.h>
#include <gsl/gsl_sf_bessel.h>

int main(int argc, char **argv)
{
    printf("%g\n", gsl_sf_bessel_Jn(atoi(argv[1]), atof(argv[2])));
    return 0;
}
```
and `./foo 1 3.4` correctly prints `0.179226`